packages/access-decision-manager-express/examples/blog-post--role-based-rbac/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "examples-blog-post",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/app.ts",
+  "scripts": {
+    "tsc": "tsc",
+    "dev": "ts-node-dev --respawn --transpileOnly ./src/app.ts",
```

Comment:
I would name this `start` instead of `dev` so the `npm start` command works.

Personally I think it is best to optimize for the developer experience and to make that as easy to start as possible (`npm start` being the obvious thing to try first if you don't read the `README.md` file (which is currently missing)).  If you need a production script that is different, that is usually going to be automated, and so it's easy to create a `npm run start:production` script for that.

Reply:
You're totally right ... updating!